boot/zephyr/pm.yml

```diff
@@ -1,33 +1,38 @@
-# The size of this partition is defined by the kconfig symbol
-# CONFIG_PM_PARTITION_SIZE_MCUBOOT
+#include <autoconf.h>
```

Comment:
We discussed automagically injecting these in the linker command, so that the .yml files does not have to have this line.

How do you feel about doing that? IMO it is easier to simply explain to the user that KCONFIG is available here, than to say that it is preprocessed AND that you can include any header file AND that this is the header file you want if you want kconfig options.

Reply:
I'm on the fence about it. The way shown here is more transparent. I get your point about explaining it, but if they are using Zephyr they are probably familiar with preprocessing, so it isn't a big jump. We anyway need to explain somewhere that it is preprocessed, or else they might get other mysterious surprises.

Also, auto-including needs a separate build-step, with an additional intermediate file, so this is simpler in that sense as well.

Reply:
I agree, magic can be dangerous :)